parlai/mturk/tasks/woz/make_scenarios.py

```diff
@@ -15,59 +16,153 @@
 db_dir = 'templates/dbs/'
 scenarios_per = 5
 
+
 def sample(parameter):
-  if parameter.get("Type") == "Integer":
-      _min = parameter.get("Min", 0)
-      _max = parameter.get("Max", 100)
-      return random.randint(_min, _max)
-  elif parameter.get("Type") == "Categorical":
-      return random.choice(parameter.get("Categories"))
-  elif parameter.get("Type") == "Boolean":
-      return random.choice([False, True])
-
-def populate(desc, db_path):
-  db = json.load(open(db_path))
-  db_params = {e['Name']: e for e in db}
-
-  # Detect all slots
-  slots = [s[:-1] if s[-1] in string.punctuation else s 
-           for s in re.split(r"[ '(]", desc) if s.startswith('@')]
-  slots = list(set(slots))
-
-  # Sample slots
-  slot_values = {}
-  param_selected = defaultdict(list)
-  for slot in slots:
-    param = slot.split('-')[0][1:]
-    value = sample(db_params.get(param))
-
-    # Resample if already selected
-    while value in param_selected[param]:
-      value = sample(db_params.get(param))
-
-    param_selected[param].append(value)
-    slot_values[slot] = value
-    
-  # Replace description
-  for slot,value in slot_values.items():
-    desc = desc.replace(slot, str(value))
-
-  return desc
+    if parameter.get("Type") == "Integer":
+        _min = parameter.get("Min", 0)
+        _max = parameter.get("Max", 100)
+        return random.randint(_min, _max)
```

Comment:
I fixed this in my last PR, not sure if something went wrong in the merge. You should change this to `randint(int(_min), int(_max))`

Reply:
Sorry, that change wasn't merged into here yet. Updated it now.